package-lock.json

```diff
@@ -1,13 +1,13 @@
 {
     "name": "aws-toolkit-vscode",
-    "version": "1.11.0-SNAPSHOT",
```

Comment:
Accidental inclusion? it's dropping the version?

Reply:
Weird, might be. I'll give it a look.